website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,221 @@
+
+**Links**
+
+- [Updated debugging docs](https://reactnative.dev/docs/debugging)
+- [Announcement talk at React Universe Conf](https://www.youtube.com/watch?v=OwivVpg6Luc)
+- [Feedback thread and FAQs](https://github.com/react-native-community/discussions-and-proposals/discussions/819)
+
+### Box Shadow and Filter style props
+
+We’ve added two New-Architecture-only style props in 0.76 - `boxShadow` and `filter`. Both of these props exist on the web, and the team tried to stick to the specs when possible so that these props are predictable, familiar, and ultimately easier to adopt. As a result you can look up the web documentation to fully understand how these work, but there are a few important differences which are noted below.
+
+#### `boxShadow`
+
+`boxShadow` adds a shadow to an element, with the ability to control the position, color, size, and blurriness of the shadow. Check out the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow) for a full overview of each of these arguments, along with a fiddle to try them out for yourself. Below are a few examples of the shadows you can make.
+
+![New boxShadow style prop](../static/blog/assets/0.76-boxshadow-example.png)
+
+`boxShadow` can take either a string, which mimics the CSS syntax, or JS objects which can embed variables. For example the string `‘5 5 5 0 rgba(255, 0, 0, 0.5)’` and the object `{offsetX: 5, offsetY: 5, blurRadius: 5, spreadDistance: 0, color: ‘rgba(255, 0, 0, 0.5)’}` would yield the same box shadow.
+
+The difference between `boxShadow` and the [existing shadow functionality](https://reactnative.dev/docs/shadow-props) is existing shadows:
+
+- Do not work on Android
+- Cannot be [inset](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#inset)
+- Do not have [spread](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#length)
+- Do not work on `View`s if there is no background color
+- Are separate props, so they cannot share web syntax
+
+**Limitations & Spec Deviations**
+
+- The default shadow color is black, not the parent’s color
+- Android normal shadows are supported on **Android 9+**
+- Android inset shadows are supported on **Android 10+**
+
+#### `filter`
+
+`filter` adds certain graphical filters to an element. There are a mix of color filters that let you modify things like brightness, saturation, and hue as well as non-color filters that let you add blurs and shadows. Check out the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) for a full overview of each individual filter function, along with a fiddle to try them out for yourself. Below is a hot dog image with various filters applied to it.
+
+![From left to right: no filter, saturate filter, blur filter, invert filter](../static/blog/assets/0.76-filter-example.png)
+
+<p align="center">From left to right: no filter, `saturate` filter, `blur` filter, `invert` filter</p>
```

Comment:
Formatting nit, we can use `figure` and `figcaption` to improve the display a bit:

```suggestion
<figure>
  <img alt="Filters demonstration" src="/blog/assets/0.76-filter-example.png" />
  <figcaption>From left to right: no filter, `saturate` filter, `blur` filter, `invert` filter</figcaption>
</figure>
```

![Screenshot 2024-10-15 at 15 09 07](https://github.com/user-attachments/assets/d0ad52d4-9717-4536-a3a0-3dd020584336)